datastore/query.go

```diff
@@ -745,6 +745,9 @@ func (c *Client) Run(ctx context.Context, q *Query) *Iterator {
 
 // RunAggregationQuery gets aggregation query (e.g. COUNT) results from the service.
 func (c *Client) RunAggregationQuery(ctx context.Context, aq *AggregationQuery) (AggregationResult, error) {
```

Comment:
The build fails with err undefined on line 749
To define err, add named return values.

Reply:
Done.